Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -31,7 +31,6 @@ public class ChirperAccountState
         public string UserAlias { get; set; }
     }
 
-    [Reentrant]
```

Comment:
Just to make sure that either of you guys remember to copy the explanation to the samples too. :)